lib/cocoapods/installer/project_cache/target_cache_key.rb

```diff
@@ -71,7 +71,7 @@ def to_h
           key_hash
         end
 
-        # @return [String]
+        # @return [Object]
```

Comment:
It was a warning in my IDE and changed it because:
```ruby
        # @return [Hash{String => Object}]
        #         The hash containing key-value pairs that identify the target.
        #
        attr_reader :key_hash
```

I guess I could just add `, String` there too?

Reply:
bah reverted it